warrior/WarriorCore/Classes/kw_driver_class.py

```diff
@@ -157,33 +158,65 @@ def get_mandatory_arguments(self):
             args.remove('self')
         return args
 
+    def get_credential_value(self, arg, system):
+        """get the value of arg in test data file corresponding to system
```

Comment:
It is data_file, right?(typo)

Reply:
Doc string says that this method is to get the value from the 'test data file' but suppose to be 'data file'.

Reply:
Completed.